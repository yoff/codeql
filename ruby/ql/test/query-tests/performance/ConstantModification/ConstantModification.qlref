query: queries/performance/ConstantModification.ql
postprocess: utils/test/InlineExpectationsTestQuery.ql