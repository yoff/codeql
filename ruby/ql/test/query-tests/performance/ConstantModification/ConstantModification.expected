| constant_modification.rb:6:9:6:41 | call to some_attribute= | Bad monkey! |
| constant_modification.rb:15:9:15:42 | call to []= | Bad monkey! |
